aerogear/graphback
# Add path config

## Motivation
#294 

## Solution

Paths for `model` `schema` `generatedResolvers` and `customResolvers` can be configured in graphback.json with default values being the same as they were. 

## Verification

1. `graphback init <name>` to create graphback project
2. navigate to project and run `graphback generate`
3. graphback should work as usual
4. create new folder in you graphback project called `foo` and move models to `foo` folder
5. in `graphback.json` change model path to `/foo/model/` and `graphback generate` and `graphback db` should work as usual
6. in `graphback.json` change the output location for resolvers and run `graphback generate` resolvers should be generated into a new path
7. in `graphback.json` change the output of `client` and run `graphback generate`, `client` queries should be generated in a specified path
8. in `graphback.json` change output path of `schema`, `graphback generate` should create schema in a new folder
9. if any of new folders don't exist a folder should be created
10. if folder already exists it should not be overwritten but instead, files generated should be added to existing foler.
